charts/thoras/templates/collector/secret.yaml

```diff
@@ -10,3 +10,14 @@ data:
 kind: Secret
 metadata:
   name: thoras-elastic-password
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: thoras-timescale-password
+data:
+  {{- $secretObj := (lookup "v1" "Secret" .Release.Namespace "thoras-timescale-password") }}
+  {{- $secretData := (get $secretObj "data") | default dict }}
+  {{- $esPassword := (get $secretData "password") | default ((.Values.metricsCollector.timescale.password | default $newEsPassword) | b64enc) }}
```

Comment:
[boulder] It looks like we're reusing the `newEsPassword` that's declared at the top of the file for the ES secret yaml document, so in the use-case where someone installs Thoras from scratch and doesn't specify new passwords (one of the most likely scenarios),  Elasticsearch and Postgres will get set assigned the same randomly generated password. That seems like something we should avoid if it's easy to do

Can we just declare a new dedicated random password variable that's independent from `newEsPassword` in the above document? Like `{{- $newPgPassword := randAlphaNum 16 }}` or something in between line 13 and 14 -- just repeating the pattern . Shouldn't require much thought

I"d also highly recommend changing `esPassword` we use here to `pgPassword` or something to prevent confusion or weird variable collisions from the ES yaml document above this one in the same file

Reply:
Done!